modules/indexing/src/test/java/org/apache/ignite/internal/processors/database/IgnitePersistentStoreSchemaLoadTest.java

```diff
@@ -271,28 +247,81 @@ private CountDownLatch checkpointLatch(IgniteEx node) {
      * @param schema Schema name.
      */
     private void makeDynamicSchemaChanges(IgniteEx node, String schema) {
-        node.context().query().querySqlFields(
-            new SqlFieldsQuery("create index \"my_idx\" on \"Person\" (\"id\", \"name\")").setSchema(schema), false)
-                .getAll();
+        runSql("create index \"my_idx\" on \"Person\" (\"id\", \"name\")", node, schema);
 
-        node.context().query().querySqlFields(
-            new SqlFieldsQuery("alter table \"Person\" add column (\"age\" int, \"city\" char)")
-            .setSchema(schema), false).getAll();
+        runSql("alter table \"Person\" drop column " +
+            "\"city\", \"str1\", \"str2\", \"num1\", \"num2\"", node, schema);
 
-        node.context().query().querySqlFields(
-            new SqlFieldsQuery("alter table \"Person\" drop column \"city\"").setSchema(schema), false)
-            .getAll();
+        runSql("alter table \"Person\" add column (" +
+                "\"rate\" decimal(10, 2) not null," +
+                "\"str1\" char(10) not null," +
+                "\"str2\" varchar," +
```

Comment:
seems TIMESTAMP(5,1) already have precision but it not processed for now, probably some other types are not covered too ...

Reply:
I tried creating timestamp/float with custom precision/scale but it looks like we don't support (ignoring) precision/scale for TIMESTAMP/FLOAT.

For the following table
```
CREATE TABLE Person(ID INTEGER PRIMARY KEY, TIME1 TIMESTAMP, TIME2 TIMESTAMP(10,3), TIME3 FLOAT(5))
```

the following query

```
SELECT * FROM sys.table_columns WHERE column_name LIKE 'TIME%'
```

 gives the following results:

 column | table | schema | affinity | incriment | default | nullable | PK | precision | scale | type
 ---------------------------------------------------------------------------------------------------
TIME1 | PERSON | PUBLIC | false | false | null | true | false | -1 | -1 | java.sql.Timestamp
TIME2 | PERSON | PUBLIC | false | false | null | true | false | -1 | -1 | java.sql.Timestamp
TIME3 | PERSON | PUBLIC | false | false | null | true | false | -1 | -1 | java.lang.Float

Thus, they are not affected by this issue.

Reply:
I added a timestamp/float into a test to avoid possible confusions in the future.

Reply:
After discussion, the test changes were reverted back :roll_eyes: .